app/code/Dev/ProductComments/Api/Data/ItemInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
```

Comment:
missed strict type
please update your PhpStorm php file and class templates to include https://prnt.sc/qbhw9c
declare(strict_types=1);